ecorise_dashboard.py

```diff
@@ -272,13 +314,13 @@ def make_groupby_pie_chart(df,col, textinfo = None, groupby_column = 'Organizati
     fig.update_layout(showlegend=showlegend,height=250,   margin=dict(l=20, r=20, t=0, b=0))
     return fig
 
-def make_bar(df,xaxis,yaxis,label, orientation='h', textposition='inside', marker_color=eco_color_desc):
-    fig = px.bar(df, x=xaxis, y=yaxis, orientation=orientation, text=label)
+def make_bar(df,xaxis,yaxis,label, pg_count, orientation='h', textposition='auto', marker_color=eco_color_desc):
+    fig = px.bar(df, x=xaxis, y=df.index.values, orientation=orientation, text=label, title="Environmental Themes Associated with Programs ({})".format(pg_count))

```

Comment:
With things like labels and titles, it's a good idea to expose them as function arguments, even if you don't plan on changing them. It provides and "escape hatch" in case you want to reuse this function with a different title without requiring the user to edit the source code:

```suggestion
def make_bar(df,xaxis,yaxis,label, pg_count, orientation='h', textposition='auto', marker_color=eco_color_desc, title="Environmental Themes Associated with Programs ({})"):
    fig = px.bar(df, x=xaxis, y=df.index.values, orientation=orientation, text=label, title=title.format(pg_count))

```